smt/applications/tests/test_podi.py

```diff
@@ -134,8 +134,56 @@ def test_predict(self):
         sm.compute_pod(self.database, tol=1)
         sm.set_interp_options("KRG")
         sm.set_training_values(self.xt)
+
+        for predict_method in [sm.predict_values, sm.predict_variances]:
```

Comment:
Nice to add tests for errors! You have to take a look at [`assertRaises`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) assertion.

Reply:
Okay thank you! I'm checking unittest.testCase's methods